lib/SILOptimizer/Mandatory/MandatoryCombine.cpp

```diff
@@ -156,6 +214,12 @@ void MandatoryCombiner::addReachableCodeToWorklist(SILFunction &function) {
       ++iterator;
 
       if (isInstructionTriviallyDead(instruction)) {
+        if (EnableCanonicalizationAndTrivialDCE) {
```

Comment:
Could these be flattened into one `if`?

Reply:
This is something that is meant to be deleted soon. I think it is fine given that this is temporary. I need this for testing purposes for something else.